sdcm/tester.py

```diff
@@ -662,17 +662,21 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
             else:
                 self.init_nodes(db_cluster=db_cluster)
 
-            with temp_authenticator(db_cluster.nodes[0], "org.apache.cassandra.auth.PasswordAuthenticator"):
-                # running `set_system_auth_rf()` before changing authorization/authentication protocols
-                self.set_system_auth_rf(db_cluster=db_cluster)
-                if self.params.get('use_ldap'):
+            if self.params.get('use_ldap'):
+                with temp_authenticator(db_cluster.nodes[0], "org.apache.cassandra.auth.PasswordAuthenticator"):
+                    # running `set_system_auth_rf()` before changing authorization/authentication protocols
+                    self.set_system_auth_rf(db_cluster=db_cluster)
```

Comment:
Don't we need to do this change globally and not just for LDAP?

Reply:
Right. Fixed.